### DISCUSSION TITLE
essay/2022/02/12/ios-android

### DISCUSSION THREAD

**Discussion by giscus** [BenjaminKim/benjaminkim.github.io#247] [Comments]
# essay/2022/02/12/ios-android

항상 두 개의 핸드폰을 들고 다닙니다. 아이폰과 안드로이드.

https://jeho.page/essay/2022/02/12/ios-android.html

**Comment by gdaegeun539** [comment]
함께 읽으면 좋은 글에서 "운영체제 넘나들기"글의 링크가 예전 주소로 되어있는 것 같아요!

저는 어쩌다보니 안드로이드빠가 되어버렸는데, 아이폰도 써볼까 고민하게 되네요

**Reply by BenjaminKim** [reply]
@gdaegeun539 알려주셔서 감사합니다. 링크 수정해두었습니다.
마음을 열고 써보시면 아이폰도 분명 좋아하게 되실거에요. 😁
